Tileset.md

```diff
@@ -7,12 +7,20 @@ These tilesets are included into the level by a `tiles` and `tilegroup` entry.
 
 1. [Introduction](#introduction)
 2. [Tile Attributes](#tile-attributes)
+   * [Attribute Combinations](#attribute-combinations)
+   * [Deprecated Attributes](#deprecated-attributes)
 3. [Tile Datas](#tile-datas)
    * [Slope Types](#slope-types)
 4. [Adding Your Own Tiles](#adding-your-own-tiles)
    * [Initial Preparations](#step-1-initial-preparations)
    * [Adding Tiles](#step-2-adding-tiles)
    * [Adding Tilegroups](#step-3-adding-tilegroups)
+5. [Special Tiles](#special-tiles)
+   * [Adding Objects As Tiles](#adding-objects-as-tiles)
+6. [Deprecated Tiles](#deprecated-tiles)
+   * [Deprecating Your Old Tiles](#deprecating-your-old-tiles)
+
+<hr>
 
 Introduction
 ============
```

Comment:
Under the "Introduction" -> "Examples" section, all separators are thicker. I believe they should be thinner as to not be mixed up with the ones separating the main parts of the document (1-6).

Reply:
Turns out there's no way of creating a thin line in MD without adding a title. I tried `<hr style="height:1px;">` but it doesn't seem to support CSS. I also tried to add a title with an invisible character, it made the line but created a really large empty space before the line, so, not sure on what to do about this